lib/browser/api/browser-window.js

```diff
@@ -112,7 +112,6 @@ BrowserWindow.prototype._init = function () {
     if (isVisible !== newState) {
       isVisible = newState
       const visibilityState = isVisible ? 'visible' : 'hidden'
-      this.webContents.send('ELECTRON_RENDERER_WINDOW_VISIBILITY_CHANGE', visibilityState)
       this.webContents.emit('-window-visibility-change', visibilityState)
```

Comment:
Ideally we would also fire this when the window is occluded. I'll do that in a follow-up PR.